extendr-api/src/metadata.rs

```diff
@@ -162,20 +161,31 @@ fn write_method_wrapper(w: &mut Vec<u8>, func: &Func, package_name: &str, use_sy
 impl Metadata {
     pub fn make_r_wrappers(&self, use_symbols: bool) -> std::io::Result<String> {
         let mut w = Vec::new();
+
+        writeln!(w, "#' @name {}", self.name)?;
+        writeln!(w, "#' @docType package")?;
+        writeln!(w, "#' @useDynLib {}, .registration = TRUE", self.name)?;
+        writeln!(w, "NULL")?;
+        writeln!(w, "")?;
+        writeln!(w, "# Generated by")?;
+        writeln!(w, "#")?;
+        writeln!(w, "# cat(.Call(\"wrap__make_{}_wrappers\", use_symbols={}))\n", self.name, if use_symbols {"TRUE"} else {"FALSE"})?;
+        
         for func in &self.functions {
             write_function_wrapper(&mut w, func, self.name, use_symbols)?;
         }
 
         for imp in &self.impls {
             write_doc(& mut w, imp.doc)?;
-            writeln!(w, "#' @export")?;
+            writeln!(w, "# '@export")?;
             writeln!(w, "{} <- new.env(parent = emptyenv())\n", imp.name)?;
                         
             for func in &imp.methods {
+                // write_doc(& mut w, func.doc)?;
                 write_method_wrapper(&mut w, func, self.name, use_symbols, imp.name)?;
             }
 
-            writeln!(w, "#' @export")?;
+            writeln!(w, "# '@export")?;
```

Comment:
See comment below. Was this change on purpose? You're basically disabling the export.